Badge.md

```diff
@@ -1,3 +1,65 @@
 (Version >= 1.16.0)
 
-Please read: https://github.com/louislam/uptime-kuma/pull/1119
\ No newline at end of file
+- [Description](#description)
+- [Status badge](#status-badge)
+- [Ping and uptime badge](#ping-and-uptime-badge)
+- [Screenshots](#screenshots)
+
+
+# Description
+
+Status, ping and uptime badges, generated in node-js, no internet access required (based on [badge-maker](https://www.npmjs.com/package/badge-maker) aka. [shields.io](http://shields.io/))
+
+There are three API endpoints that return an SVG graphic for a given (public) monitor:
```

Comment:
Please include all badges, not only the ones from the intial PR
![image](https://github.com/louislam/uptime-kuma-wiki/assets/26258709/2a7359ed-117e-4375-bea3-9437ffe212c7)